graph/set/uid/uid.go

```diff
@@ -27,10 +31,10 @@ func (s *Set) NewID() int64 {
 	for id := range s.free {
 		return id
 	}
-	if s.maxID != Max {
+	if s.maxID != math.MaxInt64 {
```

Comment:
Why not keep `Max` here?

Reply:
It seems redundant; IDs are `int64` and there is already a label for the maximum `int64` value.